### PR TITLE
Mat 5406/export naming conventions

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE FHIR SERVICE

Jira Ticket: [MAT-5406](https://jira.cms.gov/browse/MAT-5406)
(Optional) Related Tickets:

### Summary
updated naming conventions of the exports

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/madiefhirservice/utils/ExportFileNamesUtil.java [hunk 1]
```diff
@@ -5,6 +5,9 @@
 public class ExportFileNamesUtil {
 
   public static String getExportFileName(Measure measure) {
+    if (measure.getModel().startsWith("QI-Core")) {
+      return measure.getEcqmTitle() + "-v" + measure.getVersion() + "-FHIR";
```
Comment (adongare) [inline src/main/java/gov/cms/madie/madiefhirservice/utils/ExportFileNamesUtil.java:9]: Should this end with FHIR4?

Comment (sb-prateekkeerthi) [reply src/main/java/gov/cms/madie/madiefhirservice/utils/ExportFileNamesUtil.java:9]: That is not generating the actual zip file name, it is being used as file name for individual files present inside zip and based on the screenshot provided and example only the zip file name should have the version not the reset of them and that main name would be originated from madie-measure in this PR: [https://github.com/MeasureAuthoringTool/madie-measure/pull/320](https://github.com/MeasureAuthoringTool/madie-measure/pull/320)

Comment (adongare) [reply src/main/java/gov/cms/madie/madiefhirservice/utils/ExportFileNamesUtil.java:9]: gotcha did not see another PR, my bad. I'll take a look. thanks!

**Review by adongare — APPROVED** [review]
